bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Cookie.java

```diff
@@ -217,16 +217,9 @@ public void writeToDirectory(File directory) throws IOException {
     }
 
     public void writeToFile (File versionFile) throws IOException {
-        FileOutputStream fos = new FileOutputStream(versionFile);
-        BufferedWriter bw = null;
-        try {
-            bw = new BufferedWriter(new OutputStreamWriter(fos, UTF_8));
+        try (FileOutputStream fos = new FileOutputStream(versionFile);
+             BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(fos, UTF_8))) {
```

Comment:
put this in a try block as well

Reply:
Thanks @sigee , are you suggest we add a try block for each closable resource here? I think that's won't be necessary. As this [doc](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) said: ```Any resource declared within a try block opening will be closed.```, so one try-block is enough. WDYT?

Reply:
I misread the code. you are right. +1 #shipit